packages/pn-personafisica-webapp/src/pages/__test__/Deleghe.page.test.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
-import * as isMobileHook from '@pagopa-pn/pn-commons/src/hooks/useIsMobile';
 import * as redux from 'react-redux';
 import { fireEvent, waitFor } from '@testing-library/react';
+
+import * as isMobileHook from '@pagopa-pn/pn-commons/src/hooks/useIsMobile';
```

Comment:
Sostituire import puntuale con @pagopa-pn/pn-commons

Reply:
ok, visto che occorre specificare il modulo per il mock dell'hook useIsMobile procedo sostituendo con:
`import * as commons from '@pagopa-pn/pn-commons';`
e quindi sul mock:
`const useIsMobileSpy = jest.spyOn(commons, 'useIsMobile');`

faccio lo stesso sugli altri files in cui ho riscontrato lo stesso problema di import relativi